### ISSUE TITLE
Add valid Unicode hexadecimal value range check for Unicode code point searches

### ISSUE THREAD

**Issue by chrissimpkins** [source-foundry/uni#6]
```go
import "unicode/utf8"
import "strconv"

i, _ := strconv.ParseInt("00ff", 16, 32)
minPrintableInt, _ := strconv.ParseInt("0020", 16, 32)

if i > utf8.MaxRune {
    // raise request value too high error message
} else if i < minPrinatableInt {
    // raise request value too low error message
}
```

**Comment by chrissimpkins** [comment]
available in `dev` branch, PR #9

**Comment by chrissimpkins** [comment]
released in v1.0.0
